lib/api/bucketGet.js

```diff
@@ -214,6 +214,8 @@ function processMasterVersions(bucketName, listParams, list) {
     xmlParams.forEach(p => {
         if (p.value && skipUrlEncoding.has(p.tag)) {
             xml.push(`<${p.tag}>${p.value}</${p.tag}>`);
+        } else if (p.tag === 'MaxKeys' && p.value >= 0) {
+            xml.push(`<${p.tag}>${escapeXmlFn(p.value)}</${p.tag}>`);
         } else if (p.value || p.tag === 'KeyCount') {
```

Comment:
We might want to group this case under the one below for `KeyCount` which looks to handle a similar case (i.e. with `tag === 'KeyCount' || tag === 'MaxKeys'`)